CHANGELOG.MD

```diff
@@ -4,9 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0-beta1] - September 27th, 2019
 Changes that have landed but are not yet released.
 - Support for running full feature tests with the SDK.
+- Support for optional parameters via optional configuration self referential functions. 
```

Comment:
Have a subheading here for `New Features`.

We've also added decision listeners and project config update listeners as new features since last release

Also we have a breaking change:
- This method `OptimizelyFactory.ClientWithOptions()` has been removed in favor of `OptimizelyFactory.Client(clientOptions ...OptionFunc)`

Reply:
Perfect! thanks!